tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -1939,7 +1959,7 @@ TEST_F(CompilerTest, CompileThenTestPdbUtilsWarningOpt) {
     CComBSTR pMainFileName;
     VERIFY_SUCCEEDED(pPdbUtils->GetMainFileName(&pMainFileName));
     std::wstring mainFileName = static_cast<const wchar_t *>(pMainFileName);
-    VERIFY_ARE_EQUAL(mainFileName, L"hlsl.hlsl");
+    VERIFY_ARE_EQUAL(mainFileName, L"." SLASH_W L"hlsl.hlsl");
```

Comment:
Shouldn't there be a + here?

Reply:
String literals can be concatenated without putting +.